spec/acceptance/trusted_external_data_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper_acceptance'
+
+context 'trusted external data' do
+  # Set up the trusted external command on the master
+  #
+  # TODO: Should this move to suite-level or even Rakefile-level
```

Comment:
I think yes. It should move to suite-level. Maybe even to the Setup rake tasks

Reply:
Moving to suite level seems fine, but I don't think rake is the right place. If the setup fails for some reason I want rspec to tell me about it and fail the run via a test failure of some sort.

Reply:
Yeah that's fine. I'll move it to suite-level then.